pkg

```diff
@@ -155,7 +155,7 @@ while getopts h78abdf:lopqvsSL opt; do
 	esac
 done
 
-PROJECTS="$(cat infrastructure/docker/build/docker-compose.yml | grep -E '^\s\s[a-zA-Z_-]+:$' | sed 's/[: ]//g')"
+PROJECTS="weasel"$'\n'"$("${COMPOSECMD[@]}" -f $COMPOSE_FILE config --services | sed '/^weasel$/d')"
```

Comment:
This looks good, but would you mind including a comment that explains why it's like that?